### PR TITLE
document OTEL_PHP_EXCLUDED_URLS

### PR DESCRIPTION
**Pull request by brettmc**

### CODE REVIEW

**Review by svrnm — APPROVED** [review]

File: content/en/docs/languages/php/sdk.md [hunk 1]
```diff
@@ -106,14 +121,15 @@ details, see the
...
+| `OTEL_PHP_DETECTORS`                 | `all`         | `env`, `host`, `os`, `process`, `process_runtime`, `sdk`, `sdk_provided`, `container` | `env,os,process`             | Resource detector selection                                                        |
+| `OTEL_PHP_AUTOLOAD_ENABLED`          | `false`       | `true`, `false`                                                                       | `true`                       | Enable/disable SDK autoloading                                                     |
+| `OTEL_PHP_LOG_DESTINATION`           | `default`     | `error_log`, `stderr`, `stdout`, `psr3`, `none`                                       | `stderr`                     | Where internal errors and warnings will be sent                                    |
+| `OTEL_PHP_INTERNAL_METRICS_ENABLED`  | `false`       | `true`, `false`                                                                       | `true`                       | Whether the SDK should emit metrics about its internal state (eg batch processors) |
```
Comment (theletterf) [inline content/en/docs/languages/php/sdk.md:130]:
```suggestion
| `OTEL_PHP_INTERNAL_METRICS_ENABLED`  | `false`       | `true`, `false`                                                                       | `true`                       | Whether the SDK should emit metrics about its internal state (for example, batch processors) |
```

File: content/en/docs/languages/php/sdk.md [hunk 2]
```diff
@@ -106,14 +121,15 @@ details, see the
...
+| `OTEL_PHP_DISABLED_INSTRUMENTATIONS` | `[]`          | Instrumentation name(s), or `all`                                                     | `psr15,psr18`                | Disable one or more installed auto-instrumentations                                |
```
Comment (theletterf) [inline content/en/docs/languages/php/sdk.md:131]:
```suggestion
| `OTEL_PHP_DISABLED_INSTRUMENTATIONS` | `[]`          | Instrumentation names, or `all`                                                     | `psr15,psr18`                | Disable one or more installed auto-instrumentations                                |
```

File: content/en/docs/languages/php/sdk.md [hunk 3]
```diff
@@ -106,14 +121,15 @@ details, see the
...
+| ------------------------------------ | ------------- | ------------------------------------------------------------------------------------- | ---------------------------- | ---------------------------------------------------------------------------------- |
+| `OTEL_PHP_TRACES_PROCESSOR`          | `batch`       | `batch`, `simple`                                                                     | `simple`                     | Span processor selection                                                           |
+| `OTEL_PHP_DETECTORS`                 | `all`         | `env`, `host`, `os`, `process`, `process_runtime`, `sdk`, `sdk_provided`, `container` | `env,os,process`             | Resource detector selection                                                        |
+| `OTEL_PHP_AUTOLOAD_ENABLED`          | `false`       | `true`, `false`                                                                       | `true`                       | Enable/disable SDK autoloading                                                     |
```
Comment (theletterf) [inline content/en/docs/languages/php/sdk.md:128]: Could you add a bit more detail in the descriptions as to what users can accomplish by fiddling with these? In a separate PR, if you prefer.

File: content/en/docs/languages/php/sdk.md [hunk 4]
```diff
@@ -97,6 +98,20 @@ $meter = OpenTelemetry\API\Globals::meterProvider()->getMeter('name', 'version',
...
 
+### Excluded URLs
+
+You can disable SDK autoloading if the request URL matches a regular expression,
+which will stop any telemetry from being generated or exported. This would
+typically be used in a shared-nothing PHP runtime like Apache or NGINX, for
+requests such as health checks.
+
+For example, the following configuration will disable telemetry for requests
+such as `https://site/client/123/info` and `https://site/xyz/healthcheck`:
```
Comment (theletterf) [inline content/en/docs/languages/php/sdk.md:109]:
```suggestion
You can disable SDK autoloading if the request URL matches a regular expression.
Matching an excluded URL prevents any telemetry from being generated or
exported. You can use this feature in a shared-nothing PHP runtime like Apache or
NGINX, for requests such as health checks.

For example, the following configuration turns off telemetry for requests
such as `https://site/client/123/info` and `https://site/xyz/healthcheck`:
```

**Review by theletterf — APPROVED** [review]
Just some nits.

**Review by chalin — APPROVED** [review]

File: content/en/docs/languages/php/sdk.md (hunk 3 above)
Comment (brettmc) [reply content/en/docs/languages/php/sdk.md]: Yes, I'll do this in another PR - it might need some reworking.

Comment (brettmc) [reply content/en/docs/languages/php/sdk.md]: done, #3959

**Pull request merged**